test/uri/test_http.rb

```diff
@@ -64,6 +64,20 @@ def test_select
       u.select(:scheme, :host, :not_exist, :port)
     end
   end
+
+  def test_authority
+    assert_equal('a.b.c',         URI.parse('http://a.b.c/').authority)
+    assert_equal('a.b.c:8081',         URI.parse('http://a.b.c:8081/').authority)
```

Comment:
The formatting here could perhaps line up, or be abandoned. 

That is, a few methods up, there's some ASCII art going on with the formatting.